src/app/app-winui3/InputSettingsPage.xaml.cpp

```diff
@@ -74,4 +85,37 @@ void InputSettingsPage::OnOrientationChanged(
     static_cast<TabletOrientation>(combo.SelectedIndex()));
 }
 
+bool InputSettingsPage::FixedWriteRadius() {
+  return gKneeboard->GetDoodleSettings().FixedDraw;
+}
+void InputSettingsPage::FixedWriteRadius(bool value) {
+  auto ds = gKneeboard->GetDoodleSettings();
+  ds.FixedDraw = value;
+  gKneeboard->SetDoodleSettings(ds);
+}
+bool InputSettingsPage::FixedEraseRadius() {
+  return gKneeboard->GetDoodleSettings().FixedErase;
+}
+void InputSettingsPage::FixedEraseRadius(bool value) {
+  auto ds = gKneeboard->GetDoodleSettings();
+  ds.FixedErase = value;
+  gKneeboard->SetDoodleSettings(ds);
+}
+unsigned int InputSettingsPage::EraseRadius() {
```

Comment:
Use `uint32_t` (both here and in the header) to unambiguously match the IDL - though they're currently equivalent.